ecf/hurricane/plots/jevs_hurricane_global_ens_tropcyc_plots.ecf

```diff
@@ -45,12 +46,12 @@ export cyc=00
 export job=jevs_hurricane_global_ens_tropcyc_plots_${cyc}
 
 #Define the directory for TC-stats file 
-export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/1.0/hurricane_global_ens/tropcyc/stats
+export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/v1.0/hurricane_global_ens/tropcyc/stats
```

Comment:
Let's utilize the version settings in `run.ver`. Please use ${evs_ver} in place of "v1.0" in COMINstats definition.